packet_service/types.lua

```diff
@@ -816,9 +816,9 @@ types.incoming[0x037] = struct({
     movement_speed_half = {0x28, bit(uint16, 12), offset=0},
     yalms_per_step      = {0x2A, bit(uint16, 9), offset=0}, -- Determines how quickly your animation walks
     state               = {0x2C, state},
-    linkshell1_red      = {0x2D, uint8},
-    linkshell1_green    = {0x2E, uint8},
-    linkshell1_blue     = {0x2F, uint8},
+    linkshell_red      = {0x2D, uint8},
+    linkshell_green    = {0x2E, uint8},
+    linkshell_blue     = {0x2F, uint8},
```

Comment:
Why did you remove the `1` from here? Isn't it only for the first linkshell? (And if it is, adjust the spacing so the `=` line up.)

Reply:
because the other 2 packets with the same data are written w/o a 1. it's either this or one and the service handles it this way since its 2 to 1.

Reply:
I would leave it as `linkshell1` and adjust it in other packets to be more correct. Some of these definitions were written before LS2 was even a thing, so it could just be an oversight.